layouts/html.hbs

```diff
@@ -143,7 +143,11 @@
     {{/babel}}
     </script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.2.10/iframeResizer.contentWindow.min.js"></script>
-    <link rel="stylesheet" type="text/css" href="{{sdkAssetUrl global_config.sdkVersion 'en' 'answers.css'}}">
+    {{#if (isRTL global_config.locale) }}
```

Comment:
nit: The link tag below this only change what is needed in if/else isRTL check instead of the whole tag like here. Maybe we should update either one to follow the other so it's more consistent. I'm fine either way!

Reply:
oh, your right. I'll update to match the convention you made below